src/renderers/shared/server/ReactDOMNodeStreamRenderer.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactDOMNodeStreamRenderer
+ */
+
+'use strict';
+
+var invariant = require('fbjs/lib/invariant');
+var React = require('react');
+var ReactDOMPartialRenderer = require('ReactDOMPartialRenderer');
+
+var Readable = require('stream').Readable;
```

Comment:
Hey yeah glad you like it - readable stream contains fixes that aren't necessarily backported to older Node versions (a new version was published today which contains fixes for the ._destroy stuff iirc). Generally it's just a good practice to pin to a specific stream version rather than relying on whatever version is in core. Hope that makes sense (: